codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowControllerTest.java

```diff
@@ -872,10 +873,10 @@ public void subTreeBytesShouldBeCorrect() throws Http2Exception {
 
         // Send a bunch of data on each stream.
         final IntObjectMap<Integer> streamSizes = new IntObjectHashMap<Integer>(4);
-        streamSizes.put(STREAM_A, 400);
-        streamSizes.put(STREAM_B, 500);
-        streamSizes.put(STREAM_C, 600);
-        streamSizes.put(STREAM_D, 700);
+        streamSizes.put(STREAM_A, (Integer) 400);
+        streamSizes.put(STREAM_B, (Integer) 500);
```

Comment:
@buchgr @normanmaurer yeah that's the behavior I was seeing.  Didn't see anything online describing that behavior, but it does fail to compile.